in.log

```diff
@@ -52,3 +52,4 @@ more output that is not properly prefixed
 [Wed Aug  9 09:31:41 2017] [websockets:warn] A message received from unknown worker connection
 [Wed Aug  2 13:49:10 2017] [28861:warn] Stopping worker 4210 immediately
 [Wed Aug 23 09:49:19 2017] [4087:error] org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
+[Sun Aug 27 18:09:23 2017] [25795:warn] AMQP connection error: No route to host
```

Comment:
why does it say "warn] …error"? Hm, I guess an error in a subcomponent can be considered a warning on top level, fine then

Reply:
Pretty sure this is just a message from the AMQP lib we use which is then processed as warning

Reply:
you guessed wrong: https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Plugin/AMQP.pm#L112

Reply:
Interesting but just one more example on how we can improve log output from openqa :) thx for the research!